### DISCUSSION TITLE
call for source code manual

### DISCUSSION THREAD

**Discussion by xpfy** [OpenRadioss/OpenRadioss#486] [Contributors]
At present, there is no relevant description of the code, which is unfavorable for users to use the program

**Comment by elequiniou** [comment]
Thanks for your interest in the project and your constructive remark.
README.md has been updated to reference OpenRadioss documentation in commit [3b0125c](https://github.com/OpenRadioss/OpenRadioss/commit/3b0125c3c2f84984786004c41330e8f961ccaee0)
"User Documentation" section is available and for contributors "Code Contributor Documentation" section is a good starting point to know more about the code structure, hoping this will be helpful.

**Reply by elequiniou** [reply]
I transformed this issue into a discussion, having in mind the OpenRadioss documentation should continue to be enhanced to facilitate future contributions.
